scripts/links_checker/test/command/links_checker_command_test.dart

```diff
@@ -95,6 +95,25 @@ void main() {
         verify(linksChecker.checkFiles([])).called(equals(1));
       },
     );
+
+    test(
+      ".run() excludes files from the analyze based on the given ignore parameter",
+      () {
+        final paths = ['file1', 'path/to/file2'];
+        final ignore = ['path/'];
+        when(fileHelperUtil.getFiles(any)).thenReturn([]);
+        when(argumentsParser.parseArgResults(any))
+            .thenReturn(LinksCheckerArguments(paths: paths, ignore: ignore));
```

Comment:
```suggestion
        const ignorePath = 'path/';
        final paths = ['file1', '${ignorePath}to/file2'];
        final ignore = [ignorePath];
        final arguments = LinksCheckerArguments(paths: paths, ignore: ignore);

        when(fileHelperUtil.getFiles(any)).thenReturn([]);
        when(argumentsParser.parseArgResults(any)).thenReturn(arguments);
```
Would it be more DRY?